docs/setup/aws.md

```diff
@@ -57,7 +57,7 @@ Your build outputs should appear in the `src/js/grapl-cdk/zips/` directory.
 There are a few CDK deployment parameters you'll need to configure before you can deploy.
 Each of these can be found in `bin/deployment_parameters.ts`:
 
-1. `deployName` (required)
+1. `GRAPL_DEPLOYMENT_NAME` (required)
```

Comment:
Is it actually required? `graplctl` accepts either an environment variable or a CLI argument for `GRAPL_REGION`, `GRAPL_DEPOLOYMENT_NAME`, `GRAPL_VERSION`, and `AWS_PROFILE`. The important thing is that it's specified _somehow_, isn't necessarily required to be an environment variable.